src/components/Display.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import PropType from 'prop-types';
+
+const Display = ({ result = '0', id }) => (
+  <div id={id}>
+    {result}
+  </div>
+);
```

Comment:
- For this part, please let us stick to what's in the requirements and keep the incoming prop to only contain the result, 
![image](https://user-images.githubusercontent.com/46115725/93668142-63634800-fa50-11ea-8a8f-efee280f3f46.png)

Reply:
Removed the id prop